### PR TITLE
OH2-289 | Add diseases list

### PR DESCRIPTION
**Pull request by SteveGT96**
See OH2-289.

### CODE REVIEW

File: src/state/diseases/reducer.ts [hunk 1]
```diff
@@ -56,7 +59,7 @@ export default produce((draft: IDiseaseState, action: IAction<any, any>) => {
     }
 
     case GET_DISEASEIPDOUT_LOADING: {
-      draft.diseasesOpd.status = "LOADING";
+      draft.diseasesIpdOut.status = "LOADING";
```
Comment (gasp) [inline src/state/diseases/reducer.ts:62]: Just for my own understanding, OPD means [OutPatient Department](https://en.wikipedia.org/wiki/Outpatient_department), what does IpdOut stand for?

**Review by gasp — APPROVED** [review]

File: src/state/diseases/reducer.ts (hunk 1 above)
Comment (mwithi) [reply src/state/diseases/reducer.ts:62]: It stands for InPatient Department (admitted patient, in wards with beds)

Comment (mwithi) [reply src/state/diseases/reducer.ts:62]: Sorry, the admission has a diagnosis at admission time, that could be even not a "disease" but just a "symptom", and a diagnosis at discharge time, that is the one that is taken into account for statistics. So in Open Hospital we have different fields and diseases allowed (or not) for each field IpdIn and IpdOut

**Comment by SteveGT96** [comment]
This PR and related one(Exam List, Operation List, ...) need [#582](https://github.com/informatici/openhospital-ui/pull/582) to be merged, in order to add the filter as you can see in the [mockup](https://w6nt6m.axshare.com/?code=373a8db2aee9ac436955a60fb179b091&id=dekkuo&p=operation).

File: src/state/diseases/reducer.ts (hunk 1 above)
Comment (SteveGT96) [reply src/state/diseases/reducer.ts:62]: That's correct.

On OH UI, you'll see OPD Diseases in the OPD Form, IpdIn Diseases in the admission form and IpdOut Diseases in the Discharge form...

**Review by axelpezzo — APPROVED** [review]

**Comment by mwithi** [comment]
There are some conflicts to be resolved

**Comment by mwithi** [comment]
There are again some conflicts to be resolved

**Review by mwithi — APPROVED** [review]
Last approval by @axelpezzo

**Pull request merged**